### PR TITLE
feat: update mentions to newspack and remove extra files

### PR DESCRIPTION
**Pull request by thomasguillot**
This PR cleans up all the mentions to 10up and replaces them with "Newspack". It also updates files like README etc...

### CODE REVIEW

**Review by dkoo — APPROVED** [review]
This looks fine to me. Do we want to replace any of the github workflows and CODEOWNERS with our own?

**Comment by thomasguillot** [comment]
@dkoo maybe 🤔
Some of the tests are failing. Should I just ignore them?

**Comment by dkoo** [comment]
I think once this changeset is merged the CI jobs will disappear.

**Pull request merged**
